recipes/ccache/all/conanfile.py

```diff
@@ -67,7 +67,8 @@ def validate(self):
             raise ConanInvalidConfiguration(f"{self.ref} requires C++ filesystem library, that is not supported by Clang 11 + libstdc++.")
 
     def build_requirements(self):
-        self.tool_requires("cmake/3.25.3")
+        self.tool_requires("cmake/[>=3.15 <4]")
+        self.tool_requires("pkgconf/1.9.3") # for zstd
```

Comment:
This doesn't work properly on Windows from what I can see.

My advice would be to add:
```
deps.set_property("zstd", "cmake_find_mode", "module")
```

further down below (around line 89), which will cause Conan to generate `Findzstd.cmake` which will be used instead, and then pkgconf is no longer needed.